PHPCI/Model/Build.php

```diff
@@ -269,13 +269,20 @@ public function getBuildPath()
      */
     public function removeBuildDirectory()
     {
-        $buildPath = $this->getBuildPath();
+        // Get the path and remove the trailing slash as this may prompt PHP
+        // to see this as a directory even if it's a link.
+        $buildPath = rtrim($this->getBuildPath(), '/');
 
         if (!$buildPath || !is_dir($buildPath)) {
             return;
         }
 
-        exec(sprintf(IS_WIN ? 'rmdir /S /Q "%s"' : 'rm -Rf "%s"', $buildPath));
+        if (is_link($buildPath)) {
```

Comment:
Alternatively, since the windows command is the  same the logic could have been:
```
if (IS_WIN) {
    ...exec windows command
} else {
    ...exec with is_link logic
}